gain_tx: 0.4
gain_ty: 0.4
gain_tz: 0.1
gain_yaw: 0.4
